src/benchmark.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+outFile=strong_scaling_results.csv
+
+repetitions=3
+
+rm -f $outFile
+
+header="threads"
+for ((r = 0; r < repetitions; r++)); do
+  header="${header},instime${r}"
+done
+
+for ((r = 0; r < repetitions; r++)); do
+  header="${header},deltime${r}"
+done
+echo "$header" | tee $outFile
+
+for i in {0..6}; do
+  t=$((2 ** i))
+  line=""
+  for ((r = 0; r < repetitions; r++)); do
+    output=$(./pcsr2 -threads=$t -update_file=../data/update_files/insertions.txt | sed '/Elapsed/!d' | sed -n '0~2p' | sed 's/Elapsed wall clock time: //g')
```

Comment:
We should try to avoid hard-coding input files paths (and similar), not a big issue but will make running different experiments easier later on - having the path as a script argument would be better.